test/zippy/mzcompose.py

```diff
@@ -155,11 +159,15 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             },
         ),
         Materialized(
-            external_minio=True,
-            external_cockroach=True,
+            external_minio="toxiproxy",
+            external_cockroach="toxiproxy",
             sanity_restart=False,
         ),
+        # Override so seed gets respected
+        Toxiproxy(seed=random.randrange(2**63)),
```

Comment:
This needs to use the seed from `parser.add_argument("--seed", metavar="N", type=int, help="Random seed", default=1)` . Ditto for the parallel-workload mzcompose?

Reply:
I don't think so. A few lines before we do `random.seed(args.seed)`

Reply:
oh, ok ack.